integrationTests/pom.xml

```diff
@@ -10,14 +10,14 @@
   </parent>
   <groupId>org.sonarsource.scanner.gradle</groupId>
   <artifactId>it-gradle</artifactId>
-  <version>3.0-SNAPSHOT</version>
+  <version>3.3-SNAPSHOT</version>
   <name>Gradle Plugin :: Integration Tests</name>
   <inceptionYear>2015</inceptionYear>
 
   <properties>
     <!-- Default version when running ITs on dev machines. Value overridden in .cirrus.yml -->
-    <gradle.version>5.6.4</gradle.version>
-    <androidGradle.version>3.3.2</androidGradle.version>
+    <gradle.version>7.0.2</gradle.version>
```

Comment:
I think it might be a bit too much to try to support Gradle 7.0 in our ITs. Many test projects are failing with errors like `Could not find method compile() for arguments [{group=commons-io, name=commons-io, version=2.5}]`
Maybe you should stick to Gradle < 7 for now.

Reply:
I downgraded to 6.8.3, as it's the latest-below-7 version, will try to resort other test cases

Reply:
Okay local runs on my machine get all the test passed, please review them one more time @henryju. Thanks so much!